app/src/components/main/grid/gridHeader/checkboxHeaderCell/checkboxHeaderCell.scss

```diff
@@ -1,11 +1,14 @@
 .checkbox-header-cell {
   display: table-cell;
-  position: relative;
+  position: sticky;
+  top: 0;
+  z-index: 2;
```

Comment:
I've added a variable to `variables/z-index-variables.scss` for sticky toolbar z-index, you can use it too

Reply:
Thank you. Fixed.